ds_judgements_public_ui/templates/includes/footer.html

```diff
@@ -1,67 +1,96 @@
+            {% url 'courts_and_tribunals' as search_url %}
+            {% trackable_link "Courts and tribunals in Find Case Law" href=search_url %}
           </li>
           <li>
-            <a class="{{ link_class_prefix }}-gazette-link" href="https://www.thegazette.co.uk/">The Gazette</a>
+            {% url 'terms_and_policies' as search_url %}
+            {% trackable_link "Terms and policies" href=search_url %}
           </li>
           <li>
-            <a class="{{ link_class_prefix }}-judicaiary-link" href="https://www.judiciary.uk">Judiciary</a>
+            {% url 'publishing_policy' as search_url %}
+            {% trackable_link "Publishing policy" href=search_url %}
           </li>
         </ul>
       </div>
       <div>
-        <h3>About</h3>
+        <h3>
+          {% url 'help_and_guidance' as search_url %}
+          {% trackable_link "Help and guidance" href=search_url %}
+        </h3>
         <ul>
           <li>
-            <a class="{{ link_class_prefix }}-about-this-service-link" href="{% url "about_this_service" %}">About this service</a>
-          </li>
-          <li>
-            <a class="{{ link_class_prefix }}-how-to-use-this-service-link" href="{% url "how_to_search_find_case_law" %}">How to search</a>
+            {% url 'how_to_search_find_case_law' as search_url %}
+            {% trackable_link "How to search" href=search_url %}
           </li>
           <li>
-            <a class="{{ link_class_prefix }}-help-and-guidance-link" href="{% url "help_and_guidance" %}">Help and guidance</a>
+            {% url 'understanding_judgments_and_decisioms' as search_url %}
+            {% trackable_link "Understanding judgments and decisions" href=search_url %}
           </li>
           <li>
-            <a class="{{ link_class_prefix }}-contact-link" href="mailto:caselaw@nationalarchives.gov.uk">Contact us by email</a>
+            {% url 'contact_us' as search_url %}
+            {% trackable_link "Contact us" href=search_url %}
           </li>
         </ul>
       </div>
+      <div>
+        <h3 class="site-footer__show-below-medium">Open Justice Licence</h3>
+        <h3 class="site-footer__hide-below-medium">Re-use Find Case Law records </h3>
+        <div>
+          <img class="site-footer__ojl-logo"
+               src="{% static 'images/Open-Justice-Licence-logo.svg' %}"
+               alt="Open Justice Licence logo"
+               height="50"
+               width="32" />
+        </div>
+        <div>
+          All content is available under the <a class="site-footer__link"
+                                                href="{% url 'open_justice_licence' %}"
+                                                rel="license">Open Justice Licence</a>.
+        </div>
+        <br/>
+        For computational analysis, which is not allowed under the OJL <a href="{% url "transactional-licence-form" %}">apply for a license</a>.
```

Comment:
_Super_ nit-picky point for @Terry-Price here, should this be:

```suggestion
        For computational analysis, which is not allowed under the OJL, <a href="{% url "transactional-licence-form" %}">apply for a license</a>.
```

to add a missing comma and properly separate the subordinate clause?